Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -551,6 +551,38 @@ theorem prod_embDomain [Zero M] [CommMonoid N] {v : α →₀ M} {f : α ↪ β}
 #align finsupp.prod_emb_domain Finsupp.prod_embDomain
 #align finsupp.sum_emb_domain Finsupp.sum_embDomain
 
+lemma mem_embDomain [Zero M] {f : α ↪ β} {g : β →₀ M} :
```

Comment:
Should this maybe be called `exists_embDomain_eq`?

Reply:
Yes.